quantumdb-core/src/main/java/io/quantumdb/core/migration/operations/DropColumnMigrator.java

```diff
@@ -30,7 +30,9 @@ public void migrate(Catalog catalog, RefLog refLog, Version version, DropColumn
 
 		List<Index> indexes = Lists.newArrayList(table.getIndexes());
 		for (Index index : indexes) {
-			table.removeIndex(index.getColumns());
+			if (index.getColumns().contains(operation.getColumnName())) {
+				table.removeIndex(index.getColumns());
+			}
```

Comment:
It would drop all indexes when a column was dropped, now it will only drop the indexes that had the column.